crates/adapters/Cargo.toml

```diff
@@ -139,6 +139,7 @@ inventory = { workspace = true }
 backtrace = { workspace = true }
 itertools = { workspace = true }
 metrics-process = { version = "2.4.0", default-features = false }
+serde_json_path_to_error = "0.1.5"
```

Comment:
I think you are supposed to always use `workspace`.
(Maybe you can fix the `metrics-process` too)